MIGRATION_GUIDE.md

```diff
@@ -2,112 +2,92 @@
 
 The following modules contain changes:
 
-1. sda-commons-server-testing
-2. sda-commons-server-auth-testing
-3. sda-commons-server-opentracing
-4. sda-commons-server-morphia
-5. sda-commons-server-kafka
+1. [sda-commons-server-testing](#1-sda-commons-server-testing)
+2. [sda-commons-server-auth-testing](#2-sda-commons-server-auth-testing)
+3. [sda-commons-server-opentracing](#3-sda-commons-server-opentracing)
+4. [sda-commons-server-morphia](#4-sda-commons-server-morphia)
+5. [sda-commons-server-kafka](#5-sda-commons-server-kafka)
+6. [Deployment and Release of upgraded services](#deployment-and-release-of-upgraded-services)
 
 ## 1 sda-commons-server-testing
 
-Does not provide any Junit 4 rules anymore. You should find Junit 5 extensions for all of
-your rules. We recommend to migrate all your Junit 4 tests to Junit 5.
+Does not provide any Junit 4 rules anymore.
```

Comment:
NIT: `JUnit` instead of `Junit` in the whole document

Reply:
fixed